README.md

```diff
@@ -102,6 +102,26 @@ make install
 
 You will now find the binary at `$GOPATH/bin/terraform-provider-libvirt`.
 
+#### Windows
+
+To build it on Windows (64bit) one can use MinGW64 (http://www.msys2.org/)
+
+Install Golang on Windows  
+Clone terraform-provider-libvirt repository  
+Open MinGW64 Console
+```console
+pacman -S mingw-w64-x86_64-libvirt
```

Comment:
is here `pacman`more archlinux specific?

Reply:
MSYS2 use pacman as packager: https://github.com/msys2/msys2/wiki/Using-packages